server/channels/app/export.go

```diff
@@ -146,8 +146,21 @@ func (a *App) BulkExport(ctx request.CTX, writer io.Writer, outPath string, job
 		}
 	}
 
-	ctx.Logger().Info("Bulk export: exporting teams")
-	teamNames, appErr := a.exportAllTeams(ctx, job, writer)
+	var teamNames map[string]bool
+	var appErr *model.AppError
+	teamId := ""
+	if opts.TeamName == nil {
+		ctx.Logger().Info("Bulk export: exporting teams")
+		teamNames, appErr = a.exportAllTeams(ctx, job, writer)
+	} else {
+		ctx.Logger().Info("Bulk export: exporting a single team")
+		team, errGet := a.Srv().Store().Team().GetByName(*opts.TeamName)
+		if errGet != nil {
+			return model.NewAppError("BulkExport", "app.team.get.app_error", nil, "team="+*opts.TeamName, http.StatusInternalServerError).Wrap(errGet)
+		}
```

Comment:
Nit: we can avoid the extra `errGet` variable if we pre declare  `team` as `var team *model.Team`